### ISSUE TITLE
Launch Open Source Pledge

### ISSUE THREAD

**Issue by chadwhitacre** [opensourcepledge/opensourcepledge.com#4]
We want to make it socially acceptable for companies to pay meaningful amounts of money to Open Source every year. We are soft-launched at 👉 [**opensourcepledge.com**](https://opensourcepledge.com/).

## Launch Timeline

| Date | Milestone |
|---|---|
| ~September 15~ | Last day to lock in billboard **logo placement** for launch |
| October 8 | Big **[marketing campaign](https://github.com/opensourcepledge/osspledge.com/issues/11)** kicks off, SF billboards, etc. |

([kick-off call meeting notes](https://docs.google.com/document/d/1SS7ZKPkrPQLIEH63ywLwzM5vBIQqeRSka1dV1Xf-Jnk/edit?usp=sharing))

## To Do

- [x] pick a domain name
- [x] #9
- [x] #10
- [x] #8
- [x] #7
- [x] #6
- [x] #37
- [x] #5
- [x] #11

## Inspo

1. https://pledge1percent.org/
2. https://www.ericholscher.com/blog/2018/mar/9/one-percent-for-open-source/
3. https://givingpledge.org/about
4. https://www.theclimatepledge.com/
5. https://www.cos.io/initiatives/top-guidelines
6. https://uberpython.wordpress.com/2011/02/12/the-python-3-wall-of-shame/
7. https://sso.tax/
8. https://stripe.com/climate
9. https://fossunited.org/foss-pledge
10. https://www.fastly.com/fast-forward &larr; design inspo; also ... member?
11. https://wordpress.org/five-for-the-future/
12. https://esr.gitlab.io/loadsharers/
13. https://www.onepercentfortheplanet.org/
14. https://www.founderspledge.com/

**Comment by chadwhitacre** [comment]
Talked to [Neon](https://neon.tech/), idea from them to partner with [SafeBase](https://safebase.io/products/trust-center) ([e.g.](https://trust.neon.tech/)) to deliver OSS Pledge attestations alongside other things like SOC2.

**Comment by chadwhitacre** [comment]
Talked to [Clerk](https://clerk.com/), positive vibe, gonna circle back when we're closer.

**Comment by nehzatarmin** [comment]
@chadwhitacre not sure if you've seen this on HN but we gave you guys a little mention :) https://news.ycombinator.com/item?id=40905849 

The positive feedback we're also getting for Sentry is consistent month on month as we get more maintainers come onboard 
<img width="494" alt="image" src="https://github.com/getsentry/osspledge.com/assets/120353582/38c0de46-e92c-4c1f-86ad-be45b99e73d6">

**Comment by chadwhitacre** [comment]
@selviano and I are on a call about this, we just signed for SF billboards in October(!), so we now have a deadline. I've split out three tickets for our high-level todo items (from the issue description):

- https://github.com/getsentry/osspledge.com/issues/6
- https://github.com/getsentry/osspledge.com/issues/5
- https://github.com/getsentry/osspledge.com/issues/7

**Comment by chadwhitacre** [comment]
Alright! @selviano and I met with design. I've added a few items to our todo.

**Comment by chadwhitacre** [comment]
We've spruced up the landing page a bit. Even though it's temporary, we wanted something a little less plain to aid with #5.

<img width="1136" alt="Screenshot 2024-07-15 at 9 44 31 PM" src="https://github.com/user-attachments/assets/08de424e-7d17-4774-8686-8b711f943057">

**Comment by stevekrouse** [comment]
This is awesome! I'm so in! I feel like we were all missing a reasonable, simple standard and this feels like it

**Comment by chadwhitacre** [comment]
Woo-hoo! Thanks for jumping in @stevekrouse!

For everyone on this thread: we are starting weekly calls to coordinate on recruiting and execution. Mondays at 11 AM US/Pacific. Drop me an email chadwhitacre@sentry.io or lmk here if you want in on the invite. Let's go! 🙌

**Comment by Delta456** [comment]
I don't have any company currently that can fund the initiative but I still want to help anyhow. I would love to get invited.

**Comment by chadwhitacre** [comment]
Thanks @Delta456! I've added you to the invite (grabbed your email from your GH profile). 👍

**Comment by jmduke** [comment]
lets gooooooooooo

**Comment by usmanmusa1920** [comment]
All your comments are good, may we see a potential progress.

**Comment by patrick91** [comment]
I'm in as well! patrick.arminio@gmail.com

**Comment by chadwhitacre** [comment]
Awesome, thanks Patrick! Added. 👍

**Comment by zanieb** [comment]
Just want to note that [Astral](https://astral.sh) is in on this initiative — we've just [announced an open source fund](https://astral.sh/blog/astral-oss-fund) at ~$3000 per developer.

**Comment by chadwhitacre** [comment]
Woo-hoo! Welcome aboard, @zanieb! Congrats on shipping the Astral OSS Fund! 😍

**Comment by anehzat** [comment]
> Just want to note that [Astral](https://astral.sh) is in on this initiative — we've just [announced an open source fund](https://astral.sh/blog/astral-oss-fund) at ~$3000 per developer.

This is great to see @zanieb bravo to the Astral team for their leadership in this space

**Comment by chadwhitacre** [comment]
Okay! We had our community organizing kick-off call, here are [the notes](https://docs.google.com/document/d/1SS7ZKPkrPQLIEH63ywLwzM5vBIQqeRSka1dV1Xf-Jnk/edit). @selviano Can you share the recording? I believe it went to your drive.

**tl;dr** is that we organized into four working groups:

- storytelling - @selviano
- member outreach - @vladh
- maintainer outreach - @Ethan-Arrowood
- design / build / specifics - @chadwhitacre

Get in touch with the people specified if you would like to help with that aspect. Each point person should run their own team. I'll set up a weekly recurring call with @selviano @vladh @Ethan-Arrowood for overall steering. Oh! We also [started a Discord](https://discord.gg/v8vuSwky), thanks to @Delta456. 😎

Thanks to everyone who joined! Lot of energy, lets go! 👐 💃

**Comment by chadwhitacre** [comment]
Here's the [recording of the kick-off call](https://drive.google.com/file/d/1SGF-eUyjE7KllThCV6-dxVIwNUi7SIwv/view), for posterity. 😌

**Comment by Delta456** [comment]
Meet link for the next weekly community call, Monday at 1:30 PM US/Eastern.

https://discord.gg/3nf82CbF?event=1268553119717916776

**Comment by chadwhitacre** [comment]
Action items from weekly steering call:

- @vladh and @chadwhitacre to onboard [early adopter cohort](https://github.com/opensourcepledge/osspledge.com/issues/6#issuecomment-2269532574)
- @selviano to firm up Nasdaq commitment
- @chadwhitacre and @Ethan-Arrowood to start planning an [ecosystem partners kick-off event](https://github.com/opensourcepledge/osspledge.com/issues/37)
- @stevenplewis to update design

Goal is next week to be ready to add a leaderboard to the homepage.

**Comment by pepicrft** [comment]
This is a fantastic idea.

**Comment by chadwhitacre** [comment]
I added "must have" and "nice to have" labels to our tickets and updated the ticket description here. I'm meeting tomorrow with the project manager for Sentry's design team to make sure we're in a good spot. My top priority is landing #45 to unblock #5.

**Comment by chadwhitacre** [comment]
We are four weeks out from our big campaign launch on October 8! We currently have ten [companies onboarded](https://osspledge.com/members/).  🎉

From here on out all of our basic systems are in place, we just need to keep executing and building momentum.

Goals for this week:

- promotion - follow up with ecosystem partners with a partner enablement package
- promotion - publish some version of the partner package on the website for self-serve help
- recruiting - keep emailing and recruiting
- recruiting - publish some self-serve docs on the website for how to help

**Comment by chadwhitacre** [comment]
We are one week out from launch! 🙌

We just reached 20 companies onboarded!

We have a verbal commit from one foundation to publish with us on October 8!

Almost there!

**Comment by vladh** [comment]
We're launching the Pledge tomorrow at 9am Eastern!!! 🎉  😄

**Comment by chadwhitacre** [comment]
That's that, folks. Pledge is launched! I'm gonna close this out, I will likely try to collect some links in a roundup on the issue description, hopefully tomorrow.

Thank you everyone for your hard work to make this happen, especially @vladh who did so much coding and outreach and administration and community engagement and general cheerleading. From myself @dcramer @selviano @Ethan-Arrowood @Delta456 and the rest of the folks here in this project THANK YOU, Vlad, for all of your inspiring effort to launch the Open Source Pledge!! 🙇‍♂️ 🙏 💃

**Comment by vladh** [comment]
Thanks so much @chadwhitacre. 😭 ❤️ 

So excited to have the Pledge launched!!! For anyone who missed it:

![nasdaq](https://github.com/user-attachments/assets/5b0ba3c7-151d-41cb-87e2-33488d409ba2)

**Comment by chadwhitacre** [comment]
We hit [trending on X](https://x.com/i/trending/1843719347795038522).

<img width="1095" alt="image" src="https://github.com/user-attachments/assets/d54f0c16-9c7f-40f6-9554-132761ec2ae3">
